api/metrics-api-jaxrs-1.1/src/main/java/org/hawkular/metrics/api/jaxrs/handler/CounterHandler.java

```diff
@@ -217,12 +220,17 @@ public Response findCounterData(
                         .toBlocking()
                         .lastOrDefault(null);
             } else {
+                if(percentiles == null) {
```

Comment:
You do not need the null check here. JAX-RS will default to an empty list if the ```percentiles`` param is not in the query string.